.tool-versions

```diff
@@ -1,2 +1,2 @@
 erlang 26.0.2
-elixir 1.17.1
+elixir 1.17.1-otp-26
```

Comment:
Context: I had 1.17.1-otp-27 already installed, which was selected by my tool manager (mise), and then I was getting confusing errors when running any mix task because Elixir needs to be compiled with an OTP less than or equal to the one you're running.